src/containers/Settings/WebPaymentsSupport.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react'
+
+export default class WebPaymentsSupport extends Component {
+  constructor(props) {
+    super(props)
+    this.supported = (typeof PaymentRequest !== 'undefined')
```

Comment:
Does this work with the server side rendering? We usually put client side stuff inside the `componentDidMount()`.